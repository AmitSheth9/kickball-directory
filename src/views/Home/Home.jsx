import React from 'react'

export default function Home() {
    return (
        <div>
            <h2>World History Kickball League</h2>
        </div>
    )
}
