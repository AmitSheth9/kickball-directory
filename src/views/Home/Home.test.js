import { render, screen } from "@testing-library/react";
import Home from "./Home";

it('should render Home', () => {
    const {container} = render(<Home />)

    expect(container).toMatchSnapshot();
});