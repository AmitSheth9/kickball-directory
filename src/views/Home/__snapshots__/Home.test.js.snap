// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Home 1`] = `
<div>
  <div>
    <h2>
      World History Kickball League
    </h2>
  </div>
</div>
`;
